docs/running-on-yarn.md

```diff
@@ -852,8 +852,8 @@ The following extra configuration options are available when the shuffle service
   <td><code>spark.shuffle.service.db.backend</code></td>
   <td>LEVELDB</td>
   <td>
-    To specify the kind of disk-base store used in shuffle service state store, supports `LEVELDB` and `ROCKSDB` now 
-    and `LEVELDB` as default value. 
+    When Yarn NodeManager recovery is enabled, this use to specify the kind of disk-base store used in shuffle 
+    service state store, supports `LEVELDB` and `ROCKSDB` now and `LEVELDB` as default value. 
```

Comment:
```suggestion
    When work-preserving restart is enabled in YARN, this is used to specify the disk-base store used in shuffle 
    service state store, supports `LEVELDB` and `ROCKSDB` with `LEVELDB` as default value. 
```

Reply:
[d8b39ef](https://github.com/apache/spark/pull/37853/commits/d8b39eff34d8e3cf2382caa377355b6e41ec1eb4) fix this

Reply:
Thanks for your correction